src/modules/core/components/Permission/Permission.css

```diff
@@ -0,0 +1,26 @@
+.wrapper {
+  display: inline-block;
+  padding: 1px 5px;
+  height: 20px;
+  border: 1px solid var(--temp-grey-blue-7);
+  border-radius: 5px;
+  cursor: pointer;
```

Comment:
Only show the pointer cursor over them if the tooltip text is set.

For this use can use the `getMainClasses` util from `~utils/css`

Reply:
This can be used, inside the component something like this:
```js
<div className={getMainClasess(styles, {}, noPointer: !infoMessage)}>...</div>
```

then create a `noPointer` class that just removes the `pointer`:
```css
.noPointer {
  cursor: default;
}
```

The way this works is that it will apply the `main` class by default, and if `infoMessage` is not set, then it will also apply the `noPointer` class